client/web/src/enterprise/batches/list/BatchChangesChangelogAlert.tsx

```diff
@@ -8,19 +8,19 @@ import styles from './BatchChangesListIntro.module.scss'
 export const BatchChangesChangelogAlert: React.FunctionComponent = () => (
     <DismissibleAlert
         className={styles.batchChangesListIntroAlert}
-        partialStorageKey="batch-changes-list-intro-changelog-3.33"
+        partialStorageKey="batch-changes-list-intro-changelog-3.34"
```

Comment:
```suggestion
        partialStorageKey="batch-changes-list-intro-changelog-3.33"
```

I think I would maybe keep this 3.33 so people who already dismissed the deprecation notice don't have to do it again? No strong preference though.

Reply:
I think I'd prefer to have it pop up again, but I also don't feel terribly strongly about it.